_posts/04-02-01-Composer-and-Packagist.md

```diff
@@ -14,21 +14,19 @@ There are already a lot of PHP libraries that are compatible with Composer, read
 
 ### How to Install Composer
 
-You can install Composer locally (in your current working directory) or globally (e.g. /usr/local/bin, recommended).
-Let's assume you want to install Composer globally:
+The safest way to download composer is by [following the official instructions](https://getcomposer.org/download/).   
+This will verify the installer is not corrupt or tampered with.  
+The installer installs Composer *locally*, in your current working directory.
```

Comment:
Weirdly, I can't see a difference in the source (in browser, Vim or Sourcetree), and If I run `pcregrep --color='auto' -n "[\x80-\xFF]"` on a local copy it doesn't appear to have any non-ASCII characters.  Any other ideas?